src/app/packages/imageryOpenLayerMap/imageryOpenLayerMap.module.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Created by AsafMas on 07/05/2017.
+ */
+import {NgModule} from '@angular/core';
+import { OpenLayerComponent } from './map/openLayer.component';
+import { RegisterToImageryProviderService } from './services/registerToImageryProviderService';
+import { ImageryModule } from '@ansyn/imagery/imagery.module';
+
+@NgModule({
+	imports: [ImageryModule],
+	declarations: [OpenLayerComponent],
+	providers: [RegisterToImageryProviderService],
+	exports: [OpenLayerComponent],
+	entryComponents: [OpenLayerComponent]
+})
+export class ImageryOpenLayerMapModule {
+	constructor(registerToImageryProviderService: RegisterToImageryProviderService) {
```

Comment:
Unnecessary service you can inject imagery provider here and set the provider data